monkey/infection_monkey/puppet/plugin_compatibility_verifier.py

```diff
@@ -20,11 +20,13 @@ def __init__(
         island_api_client: IIslandAPIClient,
         operating_system: OperatingSystem,
         exploiter_plugin_manifests: Mapping[str, AgentPluginManifest],
+        credentials_collector_plugin_manifests: Mapping[str, AgentPluginManifest] = {},
     ):
         self._island_api_client = island_api_client
         self._operating_system = operating_system
         self._plugin_manifests: Dict[AgentPluginType, Dict[str, AgentPluginManifest]] = {
-            AgentPluginType.EXPLOITER: dict(exploiter_plugin_manifests)
+            AgentPluginType.EXPLOITER: dict(exploiter_plugin_manifests),
```

Comment:
Should we build this dict elsewhere and pass it in as an argument?

Reply:
Nah, we're passing in a dictionary of exploiter manifests. I don't want to make external callers dependent on the internal data structures of this component.

Besides, this is only temporary until there are no more hardcoded plugins.